src/Qmmands/CommandService.cs

```diff
@@ -124,7 +124,11 @@ public CommandService(CommandServiceConfiguration configuration)
                 ? configuration.NullableNouns.ToImmutableArray()
                 : CommandUtilities.DefaultNullableNouns;
 
+            #if NETSTANDARD2_0
+            StringComparer = StringComparerExtensions.FromComparison(StringComparison);
```

Comment:
What is the point of this?

Reply:
.NET Standard 2.0 does not define the FromComparison method.